src/rules/rulepanel.ts

```diff
@@ -81,6 +86,20 @@ function base64encode(file: string) {
   return FS.readFileSync(file).toString('base64');
 }
 
+function renderTaintBanner(rule: ShowRuleDescriptionParams, infoImgSrc:string) {
+  if (!rule.isTaint) {
+    return '';
+  }
+  return `<div class="taint-banner-wrapper">
+            <p class="taint-banner"><span><img class="taint-info-icon" src=${infoImgSrc} alt="info"></span> 
+            This injection vulnerability was detected by the latest SonarQube or SonarCloud analysis.
```

Comment:
Nitpick: I'm not a huge fan of the "SonarQube or SonarCloud" part, it should be possible to get the information from somewhere that the server that reported the issue is SonarCloud (or not).

Reply:
> it should be possible to get the information from somewhere that the server that reported the issue is SonarCloud (or not).

I think this is kind related to SLLS-106, now the taint issues will have a field `source` which will be specific to SQ/SC, so maybe it can be used here as well